pages/belt_docs/list.mdx

```diff
@@ -377,64 +947,180 @@ let cmpU: (t('a), t('a), [@bs] (('a, 'a) => int)) => int;
 let cmp: (t('a), t('a), ('a, 'a) => int) => int;
 ```
 
-## eqU
+**Examples**
 
+```reason
+cmp(firstList, secondList, compareFn);
 ```
-let eqU: (t('a), t('a), [@bs] (('a, 'a) => bool)) => bool;
+
+Compare elements one by one `f(x, y)`. `f` returns a negative number if `x` is "less than" `y`, zero if `x`is "equal to" `y`, a positive number if `x`is "greater than" `y`.
```

Comment:
`f(x, y)` should probably be `compareFn(firstList, secondList)`?

x = `firstList`
y = `secondList`

if I am not mistaken?

Reply:
yep, should be `compareFn`! Though, `x` and `y` are "firstListX" and "secondListY", so not the whole list. Perhaps it could be worded differently?

Reply:
oh actually it's the value inside the list being compared `firstListX` and `secondListY` reads weird IMO... so maybe `x` and `y` are the best we've got there?